packages/outbox-execute/package.json

```diff
@@ -13,7 +13,7 @@
     "hardhat": "^2.9.1"
   },
   "dependencies": {
-    "@arbitrum/sdk": "^2.0.0",
+    "@arbitrum/sdk": "3.0.0-beta.5",
```

Comment:
probably would be nice if we use the migration sdk instead before mainnet migration